src/snapred/backend/data/GroceryService.py

```diff
@@ -1062,3 +1062,9 @@ def clearADS(self, exclude: List[WorkspaceName] = [], cache: bool = False):
             self.deleteWorkspaceUnconditional(workspace)
 
         self.rebuildCache()
+
+    def getADSResident(self):
```

Comment:
plural?

Reply:
**I agree that I do need a better name for this method.**  Unfortunately `getObjectNames` is an even worse name.  Maybe `getADSResidentWorkspaces`? (seems a bit long though)